python/F3DPythonBindings.cxx

```diff
@@ -15,23 +15,23 @@
 
 namespace py = pybind11;
 
-template<typename T, size_t S>
-bool load_array(const py::handle& src, bool convert, std::array<T, S>& value)
+template<typename T>
+bool load_array(const py::handle& src, bool convert, T& value)
 {
   if (!py::isinstance<py::sequence>(src))
   {
     return false;
   }
   const py::sequence l = py::reinterpret_borrow<py::sequence>(src);
-  if (l.size() != S)
+  if (l.size() != 3)
   {
     return false;
   }
 
   size_t i = 0;
   for (auto it : l)
   {
-    value[i++] = py::cast<T>(it);
```

Comment:
and any type

Reply:
then should I write a specialization or non template version of load array for f3d::vector3_t